### ISSUE TITLE
[sommelier] Failed to build

### ISSUE THREAD

**Issue by supechicken** [skycocker/chromebrew#4984]
```shell
clang-11: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
[59/62] Compiling C++ object sommelier.p/sommelier-xdg-shell.cc.o
clang-11: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
[60/62] Compiling C++ object sommelier.p/sommelier-tracing.cc.o
clang-11: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
[61/62] Compiling C++ object sommelier.p/sommelier.cc.o
clang-11: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
[62/62] Linking target sommelier
FAILED: sommelier 
clang++  -o sommelier sommelier.p/meson-generated_aura-shell-code.c.o sommelier.p/meson-generated_drm-code.c.o sommelier.p/meson-generated_gaming-input-unstable-v2-code.c.o sommelier.p/meson-generated_gtk-shell-code.c.o sommelier.p/meson-generated_keyboard-extension-unstable-v1-code.c.o sommelier.p/meson-generated_linux-dmabuf-unstable-v1-code.c.o sommelier.p/meson-generated_pointer-constraints-unstable-v1-code.c.o sommelier.p/meson-generated_relative-pointer-unstable-v1-code.c.o sommelier.p/meson-generated_text-input-unstable-v1-code.c.o sommelier.p/meson-generated_viewporter-code.c.o sommelier.p/meson-generated_xdg-shell-unstable-v6-code.c.o sommelier.p/sommelier-compositor.cc.o sommelier.p/sommelier-data-device-manager.cc.o sommelier.p/sommelier-display.cc.o sommelier.p/sommelier-drm.cc.o sommelier.p/sommelier-gtk-shell.cc.o sommelier.p/sommelier-output.cc.o sommelier.p/sommelier-pointer-constraints.cc.o sommelier.p/sommelier-relative-pointer-manager.cc.o sommelier.p/sommelier-seat.cc.o sommelier.p/sommelier-shell.cc.o sommelier.p/sommelier-shm.cc.o sommelier.p/sommelier-subcompositor.cc.o sommelier.p/sommelier-text-input.cc.o sommelier.p/sommelier-tracing.cc.o sommelier.p/sommelier-viewporter.cc.o sommelier.p/sommelier-xdg-shell.cc.o sommelier.p/sommelier.cc.o -Wl,--as-needed -Wl,--no-undefined -Wl,-O1 -Wl,--start-group -lm /usr/local/lib64/libgbm.so /usr/local/lib64/libdrm.so /usr/local/lib64/libpixman-1.so /usr/local/lib64/libwayland-client.so /usr/local/lib64/libwayland-server.so /usr/local/lib64/libxcb.so /usr/local/lib64/libxcb-composite.so /usr/local/lib64/libxcb-xfixes.so /usr/local/lib64/libxkbcommon.so -Wl,--end-group
sommelier.p/meson-generated_aura-shell-code.c.o: file not recognized: file format not recognized
clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
ninja: build stopped: subcommand failed.
sommelier failed to install: `ninja -C build` exited with 1
```

**Comment by supechicken** [comment]
Also failed with `llvm10`
```shell
[52/62] Compiling C++ object sommelier.p/sommelier-output.cc.o
clang-10: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
[53/62] Compiling C++ object sommelier.p/sommelier-tracing.cc.o
FAILED: sommelier.p/sommelier-tracing.cc.o 
clang++ -Isommelier.p -I. -I.. -I/usr/local/include/libdrm -I/usr/local/include/pixman-1 -Xclang -fcolor-diagnostics -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -O3 -flto=thin -fuse-ld=lld -D_GNU_SOURCE -DWL_HIDE_DEPRECATED '-DXWAYLAND_PATH="/usr/local/bin/Xwayland"' '-DXWAYLAND_GL_DRIVER_PATH="/usr/lib64/dri"' '-DXWAYLAND_SHM_DRIVER="noop"' '-DSHM_DRIVER="noop"' '-DVIRTWL_DEVICE="/dev/null"' '-DPEER_CMD_PREFIX="/usr/local/lib64/ld-linux-x86-64.so.2"' '-DFRAME_COLOR="#f2f2f2"' '-DDARK_FRAME_COLOR="#323639"' -MD -MQ sommelier.p/sommelier-tracing.cc.o -MF sommelier.p/sommelier-tracing.cc.o.d -o sommelier.p/sommelier-tracing.cc.o -c ../sommelier-tracing.cc
clang-10: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
../sommelier-tracing.cc:9:10: fatal error: 'memory' file not found
#include <memory>
         ^~~~~~~~
1 error generated.
[54/62] Compiling C++ object sommelier.p/sommelier-shell.cc.o
clang-10: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
[55/62] Compiling C++ object sommelier.p/sommelier-seat.cc.o
clang-10: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
[56/62] Compiling C++ object sommelier.p/sommelier-viewporter.cc.o
clang-10: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
[57/62] Compiling C++ object sommelier.p/sommelier-xdg-shell.cc.o
clang-10: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
[58/62] Compiling C++ object sommelier.p/sommelier-subcompositor.cc.o
clang-10: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
[59/62] Compiling C++ object sommelier.p/sommelier-shm.cc.o
clang-10: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
[60/62] Compiling C++ object sommelier.p/sommelier-text-input.cc.o
clang-10: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
[61/62] Compiling C++ object sommelier.p/sommelier.cc.o
clang-10: warning: argument unused during compilation: '-fuse-ld=lld' [-Wunused-command-line-argument]
ninja: build stopped: subcommand failed.
sommelier failed to install: `ninja -C build` exited with 1
```

**Comment by supechicken** [comment]
Works with #4889

**Comment by uberhacker** [comment]
Fixed by #4889.
